src/project.ts

```diff
@@ -120,6 +120,39 @@ export class ProjectModule {
             });
     }
 
+    private mapDefaultColorPalette(){
+        return DEFAULT_PALETTE.map((item: IColor) => {
+            const guid = `${item.r}_${item.g}_${item.b}`;                        
+            return {
+                guid,
+                fill:item
+            };
+        });
+    }
+
+    /**
+     * Fetches a chart color palette for a project represented by the given
+     * projectId parameter.
+     *
+     * @method getColorPalette
+     * @param {String} projectId - A project identifier
+     * @return {Array<IPaletteItem>} An array of obejcts representing a project's
+     * color palette
+     */
+    public getFullColorPalette(projectId: string):Promise<Array<IColorPaletteItem>> {
```

Comment:
`Array<IColorPaletteItem>>` I would expect here `IColorPaletteItem[]`

Reply:
Thx my VS somehow disable validation this kind of problems are there more give me sec I will fix all of them ...